Application/EdFi.Ods.Common/Infrastructure/SqlServer/SqlServerStructuredMappings.cs

```diff
@@ -0,0 +1,28 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using NHibernate.Type;
+using System;
+using System.Collections.Generic;
+
+namespace EdFi.Ods.Common.Infrastructure.SqlServer
+{
+    public class SqlServerStructuredMappings
```

Comment:
Probably makes sense to make this class `static` as well.

Reply:
Fixed, thank you.